### PR TITLE
start at zenodo metadata file

### PR DESCRIPTION
**Pull request by lwasser**
@choldgraf @betatim @mbjoseph @jlpalomino @joemcglinchy  i'd like to add you both to this license. Would you like to be included? if so please provide your orcid ID and whatever institution you'd like to credit here as a comment OR do the file directly. Thank you!!

### CODE REVIEW

**Comment by lwasser** [comment]
NOTE:  orcid.org is down so i can't access my account right now to get my actual id :)

File: .zenodo.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  "creators": [
+    {
+      "affiliation": "Earth Lab, University of Colorado - Boulder",
+      "name": "Wasser, Leah",
+      "orcid": "0000-000-000-000"
+    },
+    {
+      "affiliation": "Earth Lab, University of Colorado - Boulder",
+      "name": "Joseph, Max",
+      "orcid": "0000-000-000-000"
```
Comment (mbjoseph) [inline .zenodo.json]: 0000-0002-7745-9990

Comment (joemcglinchy) [reply .zenodo.json]: i can't seem to edit the file in-place so here is my ORCID: 0000-0003-2135-0168

Comment (lwasser) [reply .zenodo.json]: Thank you @mbjoseph  @joemcglinchy  both are now updated!!

File: .zenodo.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      "name": "McGlinchy, Joe",
+      "orcid": "0000-0003-2135-0168"
+    },
+    {
+      "affiliation": "Earth Lab, University of Colorado - Boulder",
+      "name": "Palomino, Jenny",
+      "orcid": "0000-000-000-000"
+    },
+    {
+      "affiliation": "...Berkeley",
```
[the remark this answers is not in the document]
Comment (lwasser) [reply .zenodo.json]: Thank you @choldgraf !!! :)

**Comment by lwasser** [comment]
woo hoo!! thanks all. I think we just need input from @jlpalomino (who is out until monday) and @betatim  who i know is also enjoying some holiday time!! I appreciate the quick replies. We are so close to a formal release of this package. i am super excited.

File: .zenodo.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    },
+    {
+      "affiliation": "Berkeley Institute for Data Science, UC Berkeley",
+      "name": "Holdgraf, Chris",
+      "orcid": "0000-0002-2391-0678"
+    },
+    {
+      "affiliation": "...",
+      "name": "Head, Tim",
+      "orcid": "0000-0002-000-000"
```
Comment (betatim) [inline .zenodo.json]:
```suggestion
      "orcid": "0000-0003-0931-3698"
```

File: .zenodo.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+      "name": "Palomino, Jenny",
+      "orcid": "0000-000-000-000"
+    },
+    {
+      "affiliation": "Berkeley Institute for Data Science, UC Berkeley",
+      "name": "Holdgraf, Chris",
+      "orcid": "0000-0002-2391-0678"
+    },
+    {
+      "affiliation": "...",
```
Comment (betatim) [inline .zenodo.json]:
```suggestion
      "affiliation": "Wild Tree Tech",
```

**Comment by betatim** [comment]
Thanks a lot! Slowly crawling back towards the internets after some time doing nothing :)

**Comment by lwasser** [comment]
Oh yay!! Thanks @betatim for the info!! I hope you have enjoyed the time offline!!! And thank you all for all of the hard work on earthpy.  Its pretty amazing how far its come.

**Comment by jlpalomino** [comment]
@lwasser I didn't have an ORCID already but just created one: 0000-0003-4879-9299

**Comment by lwasser** [comment]
hi team!! thank you all for the contributions to our zenodo license file. I am going to merge this as I believe we have everyone's information here now! we can always add more people as needed. i can't thank everyone enough for your contributions to earthpy. much appreciated.

**Pull request merged**